tests/test_src/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
 add_subdirectory(test_utils)
-
+add_subdirectory(test_account)
```

Comment:
👍  for splitting into separate subdirectories

Reply:
Thanks ! :smile: